about/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8"/>
+    <title>About myself</title>
+</head>
+<body>
+<h1>Обо мне</h1>
+
+<h2>Почему Front-end?</h2>
+
+<p>Примерно год назад я задумался над сменой професии. Хотелось интересной и в то же время перспективной работы.
+    Много друзей и знакомых либо уже были в IT, либо стремились туда попасть.
+    При изучении IT специальностей именно <strong>Front-end</strong> зацепил тем, что сделав что-либо ты моментально
+    видишь результат своей работы. В добавок ко всему сайты и вэб приложения прочно вошли в <em>нашу жизнь!</em></p>
+<hr/>
+<h3>Мои сильные стороны:</h3>
+<p>Очень
+    <del>ленивый</del>
+    <ins>трудолюбивый</ins>
+    <br/>
+    Хорошо знаком с химией H<sub>2</sub>O<br/>
+    Также, хорошо знаком с физикой E=MC<sup>2</sup><br/>
+    И наконец, главное качество -
+    <mark>упорство!</mark>
+</p>
+<p>Любимая цитата:</p>
+<blockquote>Иногда хватает мгновения, чтобы забыть жизнь, а иногда не хватает жизни, чтобы забыть мгновение.
+</blockquote>
+
+
+<p>Мои контакты:</p>
+<a href="https://github.com/SCR0804">GitHub</a> &#127757;<br/>
+<a href="mailto:yakimovichsergei@gmail.com">Gmail</a> &#128236;
```

Comment:
It's better to use in tags. Content without tag is a bad practice. Just put it inside `<a>`